<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dodajuser</class>
 <widget class="QDialog" name="dodajuser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj korisnika</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ipi.jpg</normaloff>../ipi.jpg</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>213</width>
     <height>115</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_user">
        <property name="text">
         <string>Korisničko ime:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_user"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_pass">
        <property name="text">
         <string>Lozinka:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_pass"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lbl_status">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_status">
        <item>
         <property name="text">
          <string>student</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>admin</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="text">
         <string>Dodaj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cncl">
        <property name="text">
         <string>Poništi</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
